<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcMakerWidget</class>
 <widget class="QWidget" name="MusicLrcMakerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>歌词制作</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>505</width>
     <height>335</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌词制作</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>335</y>
     <width>485</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="artNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>歌手名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artNameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>118</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="songNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>歌曲名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="songNameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>78</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="authorNameLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>制作者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorNameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>158</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="lrcTextEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>78</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="introductionTextEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>198</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="introductionLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>说　明：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="makeButton">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始制作</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存本地</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reviewButton">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QLabel" name="llrcIntroductionLabel">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>55</y>
     <width>400</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入着正确的歌词文本，否则会制作失败！跟着节奏按A键开始记录！</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>505</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>line_2</zorder>
  <zorder>artNameLabel</zorder>
  <zorder>artNameEdit</zorder>
  <zorder>songNameLabel</zorder>
  <zorder>songNameEdit</zorder>
  <zorder>authorNameLabel</zorder>
  <zorder>authorNameEdit</zorder>
  <zorder>lrcTextEdit</zorder>
  <zorder>introductionTextEdit</zorder>
  <zorder>introductionLabel</zorder>
  <zorder>makeButton</zorder>
  <zorder>saveButton</zorder>
  <zorder>reviewButton</zorder>
  <zorder>llrcIntroductionLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
